app/gateway/2.7.x/install-and-run/docker.md

```diff
@@ -153,61 +187,79 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
+     to store configurations in this database.
+    * All [`_LOG`](/gateway/{{page.kong_version}}/reference/configuration/#general-section)
+    parameters: set filepaths for the logs to output to, or use the values in
+    the example to  print messages and errors to `stdout` and `stderr`.
+    * [`KONG_ADMIN_LISTEN`](/gateway/{{page.kong_version}}/reference/configuration/#admin_listen):
+    The port that the Kong Admin API listens on for requests.
+    * [`KONG_ADMIN_GUI_URL`](/gateway/{{page.kong_version}}/reference/configuration/#admin_gui_url):
+    (Enterprise only) The URL for accessing Kong Manager, preceded by a protocol
+    (for example, `http://`).
+
+1. Verify your installation:
 
     Access the `/services` endpoint using the Admin API:
 
-    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/services</code></pre>
+    ```sh
+    curl -i -X GET --url http://localhost:8001/services
+    ```
 
     You should receive a `200` status code.
 
-3. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
-
-      <pre><code>http://<div contenteditable="true">{HOSTNAME}</div>:8002</code></pre>
+1. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
 
-## Install Gateway in DB-less mode
+    ```
+    http://localhost:8002
+    ```
 
-{:.important}
-> **Important**: Running {{site.base_gateway}} on Docker in DB-less mode is
-supported for both Enterprise and OSS images. However, there is currently no
-documentation for setting up an Enterprise image in DB-less mode, so adjust the
-following steps as needed for an Enterprise deploy.
+## Install Kong Gateway in DB-less mode
 
 The steps involved in starting Kong in [DB-less mode](/gateway/{{page.kong_version}}/reference/db-less-and-declarative-config) are the following:
 
-1. Create a Docker network:
+### Set up a Docker volume
 
-    This is the same as in the Pg/Cassandra guide. We're also using `kong-net` as the
-    network name and it can also be changed to something else.
+1. Create a Docker network:
 
     ```bash
     docker network create kong-net
     ```
 
+    You can name this network anything you want. We use `kong-net` as
+    an example throughout this guide.
+
     This step is not strictly needed for running Kong in DB-less mode, but it is a good
-    precaution in case you want to add other things in the future (like a rate-limiting plugin
+    precaution in case you want to add other things in the future (like a Rate Limiting plugin
     backed up by a Redis cluster).
 
-2. Create a Docker volume:
-
-    For the purposes of this guide, a [Docker Volume][Docker Volume] is a folder inside the host machine which
-    can be mapped into a folder in the container. Volumes have a name. In this case we're going
-    to name ours `kong-vol`
+1. Create a Docker volume:
```

Comment:
We don't need to create a volume. Docker supports mounting existing folders. We can remove all references to `docker volume` and use `-v $PWD:/path/to/folder` in the `docker` command to mount the current directory

Reply:
I couldn't get this working with the syntax you provided, but this is what worked:

`-v "$(pwd)"/source:/target/ ` 

So, I did: 
`-v "$(pwd)"/declarative:/declarative/ ` 

And pointing to `/usr/local` caused permissions errors, which is why I just went with `/declarative/`. Do we want to assume that users will have the knowledge to work out any permissions issues, or does it seem fine to provide a sample path like this?